.github/workflows/tests.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 15.x]
```

```suggestion
        node-version: [15.x, 16.x]
```

Working on my dev `16.1`